Hush/Assets/Scripts/Game/SceneManager.cs

```diff
@@ -26,15 +29,25 @@ public void LoadNextScene()
             LoadDevScene();
         }
 
-        public void LoadDevScene()
+        public void LoadPreviousScene()
+        {
+            UnityEngine.SceneManagement.SceneManager.LoadScene(PlayerPrefs.GetInt("CurrentLevel"));
+        }
+
+        public void LoadMainMenu()
         {
             UnityEngine.SceneManagement.SceneManager.LoadScene(0);
         }
 
-        public void LoadDemoScene()
+        public void LoadDevScene()
         {
             UnityEngine.SceneManagement.SceneManager.LoadScene(1);
         }
+
+        public void LoadDemoScene()
+        {
+            UnityEngine.SceneManagement.SceneManager.LoadScene(2);
+        }
```

Comment:
### Minor

Arguably this code can be removed since we shouldn't need to access the demo scene anymore